README.md

```
@@ -146,21 +146,52 @@ v1 hunting uses the built-in Kubernetes Events API only — flavor-agnostic, no
+pip install -e ".[test]"
 pytest -q
 ```
 
-See available commands:
+---
+
+## CLI structure
+
+Commands are nested by **provider → bucket → command**:
+
+```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)